jwst/assign_wcs/util.py

```diff
@@ -88,6 +89,70 @@ def _reproject(x, y):
     return _reproject
 
 
+def compute_scale_transform(refwcs, fiducial):
+    """Compute scaling transform
```

Comment:
Could you add a docstring indicating what `refwcs` and `fiducial` are?  I think you could probably rename `refwcs` to be just `wcs` and indicate it is a `gwcs.WCS` object.  I believe `fiducial` is a tuple of (RA, Dec)?